guides/development-workflow.md

```diff
@@ -401,27 +414,87 @@ Upgrade widget to v1.2.1
-make sure that:
-
-- we cannot see any code that could give rise to a security vulnerability
-- the story has been implemented clearly and maintainably
-- the code contains well-written commits that help us to understand the changes
-- the code follows code style guides
-- the [changelog](#tracking-changes) has been updated is the change was
-  significant
+Code reviews help us ship better quality code and provide an opportunity to
+learn from each other.
+
+### How to begin
+
+To help review code changes you can
+[enable split review within GitHub](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-comparing-branches-in-pull-requests#diff-view-options).
+
+Pull requests include a lot buttons and information that are all competing for
+our attention. There are several ways we can view the information so having a
+plan can be helpful:
+
+1. read the description to help you understand what we are doing and why. If
+   it's unclear, stopping to find out will likely lead to a more valuable review
+1. check the status is "Open" and that all "Checks" have passed. If this isn't
+   the case it may not be ready for a review. We want to review a stable set of
+   changes that aren't still being worked on. Ask the author if in doubt
+1. scan through the "Files changed" tab briefly to peak at the new state of the
+   system to get your bearings, we are looking at the general size of the change
+   as well as the different entities involved (optional)
+1. first pass (read) - Go to the the "Commit" tab, here we will see all the
+   individual commits with the oldest at the top. Go into the oldest commit and
+   start navigating using the "Next" button. Read the description and the
+   contents with the aim of understanding each change and its rationale. Make
+   quick notes to yourself for areas of interest you'd like to come back to, the
+   trick is to make these low cost so we can maintain focus. Writing thoughtful
+   feedback takes time and we may even find that the situation changes in later
+   commits, making the feedback redundant
+1. second pass (review) - Now is the time to leave feedback. Go back to the
+   oldest commit and this time as you go through think about the following,
+   leaving comments on the nearest line where appropriate:
+   - did you have a note for this commit? Now is the time to explore and test
+     any alternative ideas before making recommendations
+   - use the [following checklist](#what-to-look-for) to help assess the
+     contents
+1. submit your review and let the author know
+
+Tip: You can add `?w=1` to the end of the URL to strip our noisy whitespace.
+
+### What to look for
+
+When performing a code review we:
+
+- check the pull request is descriptive, explains the story and doesn't assume
+  knowledge. Some templates ask for screenshots to help illustrate the change
```

Comment:
Which knowledge is safe to assume? Should the description make sense to someone who knows nothing about the project? Someone who's an active contributor? Something in between? It would be good to set a baseline target audience here, I think.